packages/drivers/odsp-driver-definitions/src/sessionProvider.ts

```diff
@@ -38,6 +38,11 @@ export interface ISocketStorageDiscovery {
 	 * This is the time within which client has to refresh the session on (ODSP) relay service.
 	 */
 	refreshSessionDurationSeconds?: number;
+
+	/**
+	 * Represent the sensitivity labels info for the file.
+	 */
+	sensitivityLabelsInfo?: string;
```

Comment:
At some point it should not be optional, right? Worth having a task with all cleanup work for the future.

Reply:
Will open a task for cleanup.

Reply:
Link: https://dev.azure.com/fluidframework/internal/_workitems/edit/6877